person.rb

```diff
@@ -1 +1,42 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
 
+  def age_reader
+    @age
+  end
+
+  def name_reader
+    @name
+  end
+
+  def id_reader
+    @id
+  end
+
+  def age_writer(age)
+    @age = age
+  end
+
+  def name_writer(name)
+    @name = name
+  end
```

Comment:
- [ ] All the highlighted code is not required instead you should add a getter & setter before initialize method.

Reply:
@sja-thedude Can you please provide some example code?

Reply:
- [ ] Its just that you have to add getter & setter methods like this before initialize method.
```
attr_accessor :name, :age 
  attr_reader :id 
```

Reply:
@sja-thedude Thank you so much.